api/v1/db/json/notes-dao.js

```diff
@@ -1,9 +1,20 @@
 const _ = require('lodash');
+const appRoot = require('app-root-path');
 const fs = require('fs');
+const moment = require('moment');
 const path = require('path');
 const config = require('config');
 
-const { serializeNotes } = require('../../serializers/notes-serializer');
+const { serializeNotes, serializeNote } = require('../../serializers/notes-serializer');
+
+const { readJSONFile, writeJSONFile } = appRoot.require('/utils/fs-operations');
+
+const { dbDirectoryPath } = config.api;
+if (!fs.existsSync(dbDirectoryPath)) {
+  throw new Error(`DB directory path: '${dbDirectoryPath}' is invalid`);
+}
```

Comment:
Like we discussed in person, I think we should put this into the `fs-operations.js`. If you wanna make the change in unit test branch than feel free to ignore this comment.

Reply:
Sounds good, I'll make the change in this branch